scripts/gen.py

```diff
@@ -785,6 +807,12 @@ def generate_outfn_result_copy(dest, src):
       dest, src)
 
 
+def is_c10_out(params):
+  ptype = param_type(params[0])
+  ctype = type_core(ptype)
+  return ctype != 'Tensor' or type_is_const(ptype)
```

Comment:
I'd a small comment outlining the two cases.

Reply:
After talking to @smessmer I actually removed this check :D